libs/utils/src/serde_system_time.rs

```diff
@@ -0,0 +1,55 @@
+//! A `serde::{Deserialize,Serialize}` type for SystemTime.
```

Comment:
Expanding this comment to mention the format serialization format or changing the struct name to reflect that would be nice.

Reply:
Got it, I've improved the comment.